invoices/invoiceregistry.go

```diff
@@ -461,9 +465,36 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 	// If no action is set, cancel the htlc.
 	action := actionCancel
 
+	// Default is to not update subscribers after the invoice update.
+	updateSubscribers := false
+
 	updateInvoice := func(inv *channeldb.Invoice) error {
-		// If the invoice is already canceled, there is no
-		// further checking to do.
+		// Don't update the invoice when this is a replayed htlc.
```

Comment:
How/when would a replay of this nature happen? (same HTLC ID in particular)

Reply:
If we crash right after completing the call to `NotifyExitHopHtlc`. On restart, the htlc will be replayed.